### DISCUSSION TITLE
pymssql in SQLAlchemy

### DISCUSSION THREAD

**Discussion by andersbogsnes** [pymssql/pymssql#761] [General]
Just wondering if there's been any discussion about updating pymssql's status in SQLAlchemy, at least in the docs?

The docs mention that this project is unmaintained and should not be used, which I imagine is a big driver in adoption
https://docs.sqlalchemy.org/en/14/dialects/mssql.html#module-sqlalchemy.dialects.mssql.pymssql

**Comment by termim** [comment]
Someone who uses pymssql+SQLAlchemy could definitely start this discussion in SQLAlchemy.

**Comment by gordthompson** [comment]
pymssql was added back into SQLAlchemy's CI testing in version 2.0.5.
